packages/react/package.json

```diff
@@ -12,7 +12,8 @@
         "@fortawesome/react-fontawesome": "^0.2.0",
         "react-spring": "^9.6.1",
         "@trrack/core": "^1.0.0",
-        "d3": "^7.8.0"
+        "d3": "^7.8.0",
```

Comment:
version for d3. Is there a reason we need 7.8 or can we just do 7?

can we also consider downgrading to v5 or v6?

if that won't break anything we can choose a lower major version

Reply:
Ya I think we could theoretically go all the way down to 4, is there a reason we would want a lower version?

Reply:
Basically having a lower version means anyone who uses trrack-vis isn't forced to update their version of d3. 

e.g
If they use v6 now, they will have to update to v7.8 or end up with two trrack instances.
If downgrade to v4, their v6 version will work for us too.

I think we can't go to v4 because the stratify functions we use were added in v5. 

For now, let's keep it to v7 though. Since downgrading might require extensive testing